docs/en/user-guide/How-To/index.rst

```diff
@@ -3,6 +3,117 @@ How To
+
+- _cover_label: shows the level / the levels of the cover/s.level of the wastewater structure
+- _bottom_label: shows the level / the levels of the wastewater_node(s).bottom_level of the wastewater structure
+- _input_label: shows the levels of the reachpoints connected as to_reachpoints to one of the wastewater_nodes of the wastewater_structure
+- _output_lebel: shows the levels of the reachpoints connected as from_reachpoints to one of the wastewater_nodes of the wastewater_structure
+
+
+.. figure:: images/tww_labels.jpg
+
+
+In this example, there are 4 input-reaches to the manhole, but there are only two input-labels (I1, I2). Why?
+
+Answer: in the function, where the _input_label is calculated, can be set a filter so that only reaches with specified function_hierachic (e.g. only pwwf-reaches) are labeled (uses a specific field in the vl-channel_function_hierarchic, documentation follows).
+In the example, the 200mm-reaches are swwf-reaches and therefore not labeled.
+
+Which reach is I1, which is I2?
+
+Answer: TWW uses the azimut of the last segment of the reach (for inputs) or the first segement of the reach (for outputs) to define the order of the labels. If you stand in the North to the manhole and go clockwise around the manhole, you will first come to I1 and then later to I2.
+In the example, 300 combined-wastewater-reach is I1, 400 wastewater-reach is I2.
+
+
+How to translate a label prefix (C, B, I, O)
+--------------------------------------------
+
+To easily translate label prefixes a series of QGIS project variables have been added. If you want to change the prefixes for cover level, bottom level and entry and exit levels, change the following project settings:
+
+.. figure:: images/tww_label_prefix_settings.jpg
+
+To see your changes, you have to rerun the symbology functions manually (see next chapter)
+
+
+How to run symbology functions manually
+---------------------------------------
+
+Sometimes the labels such as bottom, cover or entry/exit levels are not correctly displayed, even if the corresponding attribut is filled in. This happens for example when you import data by INTERLIS Import or another way. May be you also decativated the triggers to speed up the import process.
+
+* Activate / Deactivate symbology triggers::
+
+   SELECT tww_sys.create_symbology_triggers()
+   SELECT tww_sys.drop_symbology_triggers()
+
+* Run **label function** for all entities (_label, _cover_label, _bottom_label, _input_label and _output_label)::
+   SELECT tww_od.update_wastewater_structure_label(NULL, true)
+
+.. figure:: images/tww_label_attributes.jpg
+
+* Run depth calculation for all entities (wastewater_structure._depth)::
+   SELECT tww_od.update_depth(NULL, true);
```

Comment:
Needs an empty line before to show the sql as code

Reply:
Thank for the corrections, is changed